### PR TITLE
UI/15407019 extension after deprecation

### PR DESCRIPTION
**Pull request by krawczykkatarzyna**
added missing extension flow ( re-enabled ) and showing correct statuses and buttons

### CODE REVIEW

File: src/components/frame/Extensions/pages/ExtensionDetailsPage/partials/ExtensionDetails/hooks.tsx [hunk 1]
```diff
@@ -44,13 +56,28 @@ export const useExtensionDetails = (extensionData: AnyExtensionData) => {
...
+  });
+
   const handleDeprecate = useCallback(async () => {
     try {
-      await deprecateAsyncFunction(deprecateExtensionValues);
+      await deprecateAsyncFunction(deprecateExtensionValues).then(() =>
+        toast.success(
+          <Toast
+            type="success"
+            title={{ id: 'extensionDetailsPage.deprecateSuccessTitle' }}
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/Extensions/pages/ExtensionDetailsPage/partials/ExtensionDetails/hooks.tsx:71]: extensionDeprecate.toast.title.success

File: src/components/frame/Extensions/pages/ExtensionDetailsPage/partials/ExtensionDetails/hooks.tsx [hunk 2]
```diff
@@ -44,13 +56,28 @@ export const useExtensionDetails = (extensionData: AnyExtensionData) => {
...
+            description={{ id: 'extensionDetailsPage.deprecateSuccessDescription' }}
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/Extensions/pages/ExtensionDetailsPage/partials/ExtensionDetails/hooks.tsx:72]: extensionDeprecate.toast.description.success

File: src/components/frame/Extensions/pages/ExtensionDetailsPage/partials/ExtensionDetails/hooks.tsx [hunk 3]
```diff
@@ -67,18 +94,35 @@ export const useExtensionDetails = (extensionData: AnyExtensionData) => {
     }
   }, [uninstallAsyncFunction, uninstallExtensionValues]);
 
+  const handleReEnable = useCallback(async () => {
+    try {
+      await reEnableAsyncFunction(reEnableExtensionValues).then(() =>
+        toast.success(
+          <Toast
+            type="success"
+            title={{ id: 'extensionDetailsPage.reEnableSuccessTitle' }}
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/Extensions/pages/ExtensionDetailsPage/partials/ExtensionDetails/hooks.tsx:103]: extensionReEnable.toast.title.success

File: src/components/frame/Extensions/pages/ExtensionDetailsPage/partials/ExtensionDetails/hooks.tsx [hunk 4]
```diff
@@ -67,18 +94,35 @@ export const useExtensionDetails = (extensionData: AnyExtensionData) => {
...
+            description={{ id: 'extensionDetailsPage.reEnableSuccessDescription' }}
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/Extensions/pages/ExtensionDetailsPage/partials/ExtensionDetails/hooks.tsx:104]: extensionReEnable.toast.description.success

File: src/components/frame/Extensions/pages/ExtensionDetailsPage/partials/ExtensionDetails/ExtensionDetails.tsx [hunk 5]
```diff
@@ -33,72 +34,68 @@ const ExtensionDetails: FC<ExtensionDetailsProps> = ({ extensionData }) => {
...
-                label={{ id: 'extensionDetailsPage.uninstallConfirmation' }}
-                onChange={() => setIsCheckboxChecked((prevState) => !prevState)}
-                classNames="mt-5"
+          <div className="mt-6 flex flex-col gap-4">
+            <div>
+              <Button mode="primaryOutline" isFullSize onClick={() => setIsReEnablelModalOpen(true)}>
+                {formatMessage({ id: 'extensionDetailsPage.reEnable' })}
+              </Button>
+              <Modal
+                title={formatMessage({ id: 'extensionDetailsPage.reEnable.modal.title' })}
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/Extensions/pages/ExtensionDetailsPage/partials/ExtensionDetails/ExtensionDetails.tsx:59]: extensionReEnable.modal.title

File: src/components/frame/Extensions/pages/ExtensionDetailsPage/partials/ExtensionDetails/ExtensionDetails.tsx [hunk 6]
```diff
@@ -33,72 +34,68 @@ const ExtensionDetails: FC<ExtensionDetailsProps> = ({ extensionData }) => {
...
+                subTitle={formatMessage({ id: 'extensionDetailsPage.reEnable.modal.subTitle' })}
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/Extensions/pages/ExtensionDetailsPage/partials/ExtensionDetails/ExtensionDetails.tsx:60]: extensionReEnable.modal.subTitle

**Review by CzarekDryl — APPROVED** [review]

**Review by KatarzynaStefanowiczPagepro — APPROVED** [review]

**Pull request merged**